src/scheduler/workers/analytics/shouldExportViewName.ts

```diff
@@ -0,0 +1,48 @@
+import { prisma } from "scheduler/workers/model";
```

Comment:
Quick question, the name of this file is `shouldExportViewName` but there are two functions being exported here. I'm curious as to why this wasn't place in a file called `helpers` or something?

I've noticed you've created files that export single functions so it would be good to know what inspired that.